core/web3/src/entitlement.ts

```diff
@@ -526,7 +526,7 @@ export function createExternalTokenStruct(addresses: Address[]) {
         return NoopRuleData
     }
     const defaultChain = addresses.map((address) => ({
-        chainId: 1n,
+        chainId: 31337n,
```

Comment:
the path of entitlement.ts suggests that this is a public function that can be used for both test and as an SDK API. I think a better treatment is to pass an optional arg in `createExternalTokenStruct(..., opts?: TokenOptions)` where this can be set by a test to 31337.

Reply:
That's a good point, Tak - these are currently only used for testing, but so is a great deal of river code and it's too much to move it all in this PR. I'll add optional overrides and refactor harmony usages.